tasks/au/geo.py

```diff
@@ -3,7 +3,7 @@
 
 from lib.timespan import get_timespan
 
-from tasks.base_tasks import (ColumnsTask, DownloadUnzipTask, Shp2TempTableTask, SimplifiedTempTableTask, TableTask,
+from tasks.base_tasks import (ColumnsTask, DownloadUnzipTask, GeoFile2TempTableTask, SimplifiedTempTableTask, TableTask,
```

Comment:
There are still 4 mentions of `Shp2TempTableTask`, two minor ones at `example.rst` and `development.rst`, and two slightly more important at `test_tabletasks.py`. I'd change all of them, as they become confusing.